Lib/test/support/__init__.py

```diff
@@ -1542,9 +1585,12 @@ def get_socket_conn_refused_errs():
 
 
 @contextlib.contextmanager
-def transient_internet(resource_name, *, timeout=30.0, errnos=()):
+def transient_internet(resource_name, *, timeout=_NOT_SET, errnos=()):
```

Comment:
Why this special value rather than `None`, as in the other cases? Is it to support `None` as an actual timeout value?

Reply:
There is one test_socket test which requires timeout=None.